src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.Managed.cs

```diff
@@ -181,6 +171,25 @@ private void PrepareTouchScroll(object sender, ManipulationStartingRoutedEventAr
 			}
 		}
 
+		private void TouchScrollStarted(object sender, ManipulationStartedRoutedEventArgs e)
+		{
+			if (e.Container != this)
+			{
+				// This gesture is coming from a nested element, we just ignore it!
+				return;
+			}
+
+#if HAS_UNO_WINUI
+			if (e.PointerDeviceType == global::Microsoft.UI.Input.PointerDeviceType.Touch)
+#else
+			if (e.PointerDeviceType == global::Windows.Devices.Input.PointerDeviceType.Touch)
+#endif
```

Comment:
Please, to improve readability keep this in the `using`. If adding the full namespace causes some issue, you should alias the type (in that case we usually prefix it with an `_` in uno):

```csharp
#if HAS_UNO_WINUI
using _PointerDeviceType = global::Microsoft.UI.Input.PointerDeviceType;
#else
using _PointerDeviceType = global::Windows.Devices.Input.PointerDeviceType;
#endif
```